### ISSUE TITLE
Loading only the newsletter section

### ISSUE THREAD

**Issue by DreamsOfImran** [Satriaana/Satriaana-Web-Site#101]
While visiting https://satriaana.github.io/Satriaana-Web-Site/#/
It's loading only the newsletter section.

**Comment by DreamsOfImran** [comment]
Created PR for it. Please review.
#100

**Comment by nimeshkavinda** [comment]
Hi @DreamsOfImran,

I have reviewed your pull request. Great work!

Thank you so much for your generous contributions to the GitHub issues regarding the website of Satriaana. We highly appreciate all the work you’ve done. Your efforts in developing our website are very commendable.

It’s with great pleasure that we’d like to inform you that we've decided to give you the chance of joining us as volunteers, after seeing your enthusiasm for open-source. So, if you’re interested, simply fill out the form through the link we’ve attached forthwith.

Note: We’re starting our volunteer onboard process in three weeks time from now on and we’re sending this form only to you as you have contributed to our website. Hence we’ll be closing the link of the form right after you’ve submitted the form. So, we would like to kindly request you to send us a message in reply after you’ve filled it.

Volunteer Form: https://satriaana.typeform.com/to/XYGqfnkI

Hope we can work together again very soon!
